src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -566,7 +566,7 @@ export default class Aircraft {
      * @method pushHistory
      */
     pushHistory() {
-        this.history.push([this.position[0], this.position[1]]);
+        this.history.push([this.position.relativePosition[0], this.position.relativePosition[1]]);
```

Comment:
`this.position.relativePosition` this little guy is used _a lot_ throughout this file.  why not create a getter for it?
```javascript
get relativePosition() {
    return this.position.relativePosition;
}
```